tests/minimal-linux.pkr.hcl

```diff
@@ -0,0 +1,182 @@
+}
+
+variable "aws_subnet_id" {
+  type    = string
+  default = ""
+}
+
+variable "aws_temporary_security_group_source_cidrs" {
+  type    = list(string)
+  default = ["0.0.0.0/0"]
+}
+
+variable "spel_amiutilsource" {
+  type    = string
+  default = env("SPEL_AMIUTILSOURCE")
+}
+
+variable "spel_disablefips" {
+  type    = string
+  default = ""
+}
+
+variable "spel_identifier" {
+  type    = string
+  default = env("SPEL_IDENTIFIER")
+}
+
+variable "spel_pypi_url" {
+  type    = string
+  default = "https://pypi.org/simple"
+}
+
+variable "spel_version" {
+  type    = string
+  default = env("SPEL_VERSION")
+}
+
+source "amazon-ebs" "base" {
+  ami_description             = "This is a validation AMI for ${var.spel_identifier}-${source.name}-${var.spel_version}.x86_64-gp2"
+  ami_name                    = "validation-${var.spel_identifier}-${source.name}-${var.spel_version}.x86_64-gp2"
+  associate_public_ip_address = true
+  communicator                = "ssh"
+  ena_support                 = true
+  force_deregister            = true
+  instance_type               = "m4.large"
+  launch_block_device_mappings {
+    delete_on_termination = true
+    device_name           = "/dev/sda1"
+    volume_size           = 21
+    volume_type           = "gp2"
+  }
+  max_retries                           = 20
+  region                                = var.aws_region
+  skip_create_ami                       = true
+  skip_save_build_region                = true
+  sriov_support                         = true
+  ssh_interface                         = var.aws_ssh_interface
+  ssh_port                              = 22
+  ssh_pty                               = true
+  ssh_username                          = "spel"
+  subnet_id                             = var.aws_subnet_id
+  tags                                  = { Name = "" } # Empty name tag avoids inheriting "Packer Builder"
+  temporary_security_group_source_cidrs = var.aws_temporary_security_group_source_cidrs
+  user_data_file                        = "${path.root}/userdata/validation.cloud"
+}
+
+build {
+  source "amazon-ebs.base" {
+    source_ami = var.aws_source_ami_centos7_hvm
+    name       = "minimal-centos-7-hvm"
+  }
+
+  source "amazon-ebs.base" {
+    source_ami = var.aws_source_ami_centos8stream_hvm
+    name       = "minimal-centos-8stream-hvm"
+  }
+
+  source "amazon-ebs.base" {
+    source_ami = var.aws_source_ami_rhel7_hvm
+    name       = "minimal-rhel-7-hvm"
+  }
+
+  source "amazon-ebs.base" {
+    source_ami = var.aws_source_ami_rhel8_hvm
+    name       = "minimal-rhel-8-hvm"
+  }
+
+  provisioner "shell" {
+    execute_command = "{{ .Vars }} sudo -E /bin/sh -ex -o pipefail '{{ .Path }}'"
+    inline = [
+      "growpart /dev/xvda 2",
+      "partprobe",
+    ]
+  }
+
+  provisioner "shell" {
+    execute_command = "{{ .Vars }} sudo -E /bin/sh -ex -o pipefail '{{ .Path }}'"
+    inline = [
+      "mkdir -p /tmp/spel/tests",
+      "chown -R spel:spel /tmp/spel",
```

Comment:
I'm having a hard time sussing-out where the `spel` user/group are/were created?

Reply:
In userdata! With cloud-config! ❤️

Reply:
That'd be why, then! =)